fulp_modules/container_init/vending_init.dm

```diff
@@ -0,0 +1,3 @@
+/obj/machinery/vending/wardrobe/engi_wardrobe/Initialize()
```

Comment:
digi shoes was merged, this should be moved to the .dm file it created.

Reply:
How?